bap-vibes/src/arm_selector.ml

```diff
@@ -920,7 +958,7 @@ struct
       end
     | Var v -> begin
         match Var.typ v with
-        | Imm 1 when Option.is_some branch ->
+        | Imm _ when Option.is_some branch ->
```

Comment:
Truth values can be any bitwidth now?

Reply:
This is how it's being translated from the C semantics. Perhaps in `Core_c` we could be a bit more strict.